docs/schema/snitch_cluster.schema.json

```
@@ -416,6 +416,12 @@
                                     "description": "Enable Smallfloat vector extension (SIMD).",
                                     "default": false
                                 },
+                                "snax": {
```

```suggestion
                                "xsnax": {
```
Seems useful to see this as a riscv instruction extension

---

@JosseVanDelm Is it better to be seen this way? Since we are technically using CRSs instead. I have a feeling that people may get confused if we term it is a RISCV instruction extension where most of what PULP did was to literally add extra instructions.

---

I don't know, the SSRs technically don't add any instructions IMHO?

---

@JosseVanDelm SSRs have their own extensions 🚀 

https://pulp-platform.github.io/snitch_cluster/rm/custom_instructions.html

SSRs are enabled via CSR instruction, BUT configuration is through extensions 🛩️